lua/lir/actions.lua

```diff
@@ -190,6 +190,24 @@ function actions.delete()
   actions.reload()
 end
 
+--- wipeout
+function actions.wipeout()
+  local ctx = get_context()
+  if not ctx:is_dir_current() then
+    local name = ctx:current_value()
+    local bufnr = vim.fn.bufnr(name)
```

Comment:
Maybe you should use `vim.fn.bufnr(ctx:current().fullpath)` instead of `vim.fn.bufnr(name)`.
I've tried it, and it sometimes doesn't work well with `vim.fn.bufnr(name)`.